specsscan/config/NXmpes_arpes_config.json

```diff
@@ -1,8 +1,7 @@
 {
   "/@default": "entry",
   "/ENTRY[entry]/@default": "data",
-  "/ENTRY[entry]/definition": "NXmpes_arpes",
-  "/ENTRY[entry]/title": "@attrs:metadata/entry_title",
+  "/ENTRY[entry]/title": "['@eln:/ENTRY/title', '@attrs:metadata/entry_title']",

```

Comment:
Just a small remark on the general notation: With the new pynxtools-mpes reader you can also use `/ENTRY/...` (w/o the `[entry]`) to allow entry replacement, i.e., it will add multiple entries according to the json template as handled by the reader (can also use a combination to both if special fields should be set only for specific entries).

It is not too relevant here since the mpes reader currently only supports one entry, so these two notations in this case do exactly the same. I just wanted to make you aware for possible future additions of multiple entry conversion.

Reply:
Done